lib/features/snapping/BpmnConnectSnapping.js

```diff
@@ -11,22 +11,19 @@ import {
 
 import { is } from '../../util/ModelUtil';
 
-import {

```

Comment:
Hard to see from the commit message: Is this a fix for snapping being done incorrectly? If that is the case I'd appreciate a few test cases. 

Or is this merely an adjustment to the new reconnection facilities. In that case, let's not call it a fix :wink:.